noxfiles/constants_nox.py

```diff
@@ -0,0 +1,47 @@
+"""Define constants to be used across the noxfiles."""
+from os import getenv
+
+
+def get_current_tag() -> str:
+    """Get the current git tag."""
+    from git.repo import Repo
```

Comment:
I don't see the `GitPython` package included in any of the requirements and it isn't present after installing dependencies. This should probably be included in `dev_requirements.txt`?

Reply:
I had no idea this wasn't part of the standard lib 👀  hah my bad! Will do